src/plugins/MamicoCoupling.cpp

```diff
@@ -1,64 +1,52 @@
 #include "MamicoCoupling.h"
 #include "Domain.h"
 
-void MamicoCoupling::readXML(XMLfileUnits& xmlconfig)
-{
-	return;
-}
+void MamicoCoupling::readXML(XMLfileUnits &xmlconfig) { return; }
 
-void MamicoCoupling::init(ParticleContainer* particleContainer,
-		DomainDecompBase* domainDecomp, Domain* domain)
-{
+void MamicoCoupling::init(ParticleContainer *particleContainer,
+                          DomainDecompBase *domainDecomp, Domain *domain) {
 #ifdef MAMICO_COUPLING
```

Comment:
Why is not the entire plugin optionally compiled?!

Reply:
Plugin now optionally compiled